<!DOCTYPE HTML>



<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  <title>Observation-genetics-cg-prf-1a - FHIR v0.4.0</title>

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="author" content="http://hl7.org/fhir"/>

  <link href="fhir.css" rel="stylesheet"/>
  <link rel="Prev" href="http://hl7.org/implement/standards/fhir/observation-genetics-cg-prf-1a"/>

  <!-- Bootstrap core CSS -->
  <link href="./dist/css/bootstrap.css" rel="stylesheet"/>
  <link href="./assets/css/bootstrap-fhir.css" rel="stylesheet"/>

  <!-- Project extras -->
  <link href="./assets/css/project.css" rel="stylesheet"/>
  <link href="./assets/css/pygments-manni.css" rel="stylesheet"/>
	<link href="jquery-ui.css" rel="stylesheet"/>

  <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!--[if lt IE 9]>
  <script src="./assets/js/html5shiv.js"></script>
  <script src="./assets/js/respond.min.js"></script>
  <![endif]-->

  <!-- Favicons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="./assets/ico/apple-touch-icon-144-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="./assets/ico/apple-touch-icon-114-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="./assets/ico/apple-touch-icon-72-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-57-precomposed.png"/>
  <link rel="shortcut icon" href="./assets/ico/favicon.png"/>

</head>
<body>
	<div id="segment-header" class="segment"><!-- segment-header -->	
		<div class="container"><!-- container -->
			<a id="logo" href="http://hl7.org/fhir"><img src="./assets/images/fhir-logo-www.png" alt="logo fhir"/> </a>
			<div id="hl7-status">
				<b>Continuous Integration Build</b>
			</div>       	
			
			<div id="hl7-nav">
				 <a id="hl7-logo" href="http://www.hl7.org">
					<img src="./assets/images/hl7-logo.png" width="42" height="50" alt="visit the hl7 website"/>
				</a> 
			</div>       	
			<!--hl7-search-->
		</div>
		<div class="container"><!-- container -->
	</div></div><!-- /segment-header -->	


	<div id="segment-navbar" class="segment"><!-- segment-navbar -->
		<div id="stripe"> </div>
		<div class="container"><!-- container -->
 <!-- HEADER CONTENT -->
 	      		
 	      		      		
			<nav class="navbar navbar-inverse">
				<div class="container">
					<button data-target=".navbar-inverse-collapse" data-toggle="collapse" class="navbar-toggle" type="button">
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
					</button>
					<a href="index.html" class="navbar-brand hidden">FHIR</a>
					<div class="nav-collapse collapse navbar-inverse-collapse">
						<ul class="nav navbar-nav">
              <li><a href="index.html">Home</a></li>
              <li><a href="documentation.html">Documentation</a></li>
              <li><a href="implementation.html">Implementation</a></li>
              <li><a href="resourcelist.html">Resources</a></li>
              <li><a href="clinical.html">Clinical</a></li>
              <li><a href="administration.html">Administrative</a></li>
              <li><a href="infrastructure.html">Infrastructure</a></li>
              <li><a href="financial.html">Financial</a></li>

						</ul>
					</div><!-- /.nav-collapse -->
				</div><!-- /.container -->
			</nav><!-- /.navbar -->
      		
				
<!-- /HEADER CONTENT -->				
		</div><!-- /container -->
	</div><!-- /segment-navbar -->
	

	<div id="segment-breadcrumb" class="segment"><!-- segment-breadcrumb -->
		<div class="container"><!-- container -->	
			<ul class="breadcrumb">
        <li><a href="index.html">Home</a></li>
        <li><a href="clinical.html">Clinical</a></li>        <li><a href="observation.html">Observation</a></li>        <li><a href="observation-genetics-cg-prf-1a.html">Profile</a></li>        <li><b>Standard Profile for Genetics</b></li>        <!-- observation-genetics-cg-prf-1a.html / profile:Observation/observation-genetics-cg-prf-1a / Standard Profile for Genetics-->

			</ul>	
		</div><!-- /container -->
	</div><!-- /segment-breadcrumb -->
	

	<div id="segment-content" class="segment"><!-- segment-content -->
	<div class="container"><!-- container -->
            <div class="row">
            	<div class="inner-wrapper">
<!-- CONTENT CONTENT -->


<div class="col-9">

<h1>Standard Profile for Genetics (Profile)</h1>



<h2>Scope and Usage</h2>
<p>
Observation-genetics-profile extends <a href="observation.html">Observation</a> resource to enable reporting of structured genetic test results. In addition, the genetics profile contextualizes well established standards from the field of clinical genetics into the standards of healthcare (e.g. HGNC - HUGO Gene Nomenclature Committee's international standard for gene names, symbols, and identifiers).

</p>




<table class="lines"><tr><td colspan="2"><b>Profiles</b>: </td></tr><tr><td><a href="observation-genetics-cg-prf-1a-genetics.html">Genetics</a></td><td>Describes how the observation resource is used to report structured genetic test results</td></tr><tr><td colspan="2"><b>Extensions</b>: </td></tr><tr><td><a name="extension-geneticsGenomeBuild"/><a href="extension-geneticsgenomebuild.html">geneticsGenomeBuild</a></td><td><b>GenomeBuild</b> : <p>The Genome Build used for reference.  Version number must be included if a versioned release of a primary build was used.</p>
</td></tr><tr><td><a name="extension-geneticsChromosome"/><a href="extension-geneticschromosome.html">geneticsChromosome</a></td><td><b>Chr</b> : <p>The chromosome containing the genetic finding - values should be 1-23, X, Y, mito, viral, bacteria.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicReferenceSequenceId"/><a href="extension-geneticsgenomicreferencesequenceid.html">geneticsGenomicReferenceSequenceId</a></td><td><b>GenomicRefSeqID</b> : <p>The genomic reference sequence identifier is included, to support testing using alternative reference sequences.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicStart"/><a href="extension-geneticsgenomicstart.html">geneticsGenomicStart</a></td><td><b>GenomicStart</b> : <p>Nucleotide location for start of genomic finding.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicStop"/><a href="extension-geneticsgenomicstop.html">geneticsGenomicStop</a></td><td><b>GenomicStop</b> : <p>Nucleotide location for end of genomic finding.</p>
</td></tr><tr><td><a name="extension-geneticsReferenceAllele"/><a href="extension-geneticsreferenceallele.html">geneticsReferenceAllele</a></td><td><b>RefAllele</b> : <p>Nucleotide(s) from genomic start to genomic stop on reference sequence. Use '-' for insertions.</p>
</td></tr><tr><td><a name="extension-geneticsObservedAllele"/><a href="extension-geneticsobservedallele.html">geneticsObservedAllele</a></td><td><b>ObsAllele</b> : <p>Oserved nucleotides from genomic start to genomic stop. Use '-' for delections.</p>
</td></tr><tr><td><a name="extension-geneticsStrand"/><a href="extension-geneticsstrand.html">geneticsStrand</a></td><td><b>Strand</b> : <p>Indicates if the variant is on the positive (+) or negative (-) strand.</p>
</td></tr><tr><td><a name="extension-geneticsGeneId"/><a href="extension-geneticsgeneid.html">geneticsGeneId</a></td><td><b>HGNCgene</b> : <p>HGNC gene symbol and identifier from HUGO Gene Nomenclature Committee at: http://www.genenames.org/.</p>
</td></tr><tr><td><a name="extension-geneticsTranscriptReferenceSequenseId"/><a href="extension-geneticstranscriptreferencesequenseid.html">geneticsTranscriptReferenceSequenseId</a></td><td><b>TransReferenceSeqID</b> : <p>Reference identifier for cDNA transcript, with version, from NCBI's RefSeq or ENSEMBL.</p>
</td></tr><tr><td><a name="extension-geneticsDNASequenceVariation"/><a href="extension-geneticsdnasequencevariation.html">geneticsDNASequenceVariation</a></td><td><b>HGVSdnaVariant</b> : <p>cDNA variant following HGVS nomenclature and identifiers to internal and external systems if known (e.g. ClinVar identifier).</p>
</td></tr><tr><td><a name="extension-geneticsVariationId"/><a href="extension-geneticsvariationid.html">geneticsVariationId</a></td><td><b>VariantId</b> : <p>Identifier for variant.  If a germline variant, ClinVar or dbSNP identifier should be used. If a somatic variant, COSMIC identifier should be used, unless in ClinVar.</p>
</td></tr><tr><td><a name="extension-geneticsDNARegionName"/><a href="extension-geneticsdnaregionname.html">geneticsDNARegionName</a></td><td><b>RegionName</b> : <p>Details of exonic location of variant (e.g. Exon 1).</p>
</td></tr><tr><td><a name="extension-geneticsDNASequenceVariationType"/><a href="extension-geneticsdnasequencevariationtype.html">geneticsDNASequenceVariationType</a></td><td><b>DNAvariantType</b> : <p>Codified type for associated DNA Sequence Variation.  DNA Sequence Variations use the HGVS notation which implies the DNA Sequence Variation Type, but the concurrent use of this code will allow a standard and explicit type for technical and display convenience. LOINC Answer List values 48019-4 or Sequence Ontology vaues.</p>
</td></tr><tr><td><a name="extension-geneticsProteinReferenceSequenceId"/><a href="extension-geneticsproteinreferencesequenceid.html">geneticsProteinReferenceSequenceId</a></td><td><b>ProteinReferenceSeq</b> : <p>Reference identifier for protein transcript, with version, from NCBI's RefSeq or ENSEMBL.</p>
</td></tr><tr><td><a name="extension-geneticsAminoAcidChange"/><a href="extension-geneticsaminoacidchange.html">geneticsAminoAcidChange</a></td><td><b>HGVSproteinChange</b> : <p>Protein variant following HGVS nomenclature and identifiers to internal and external systems if known.</p>
</td></tr><tr><td><a name="extension-geneticsAminoAcidChangeType"/><a href="extension-geneticsaminoacidchangetype.html">geneticsAminoAcidChangeType</a></td><td><b>ProteinChangeType</b> : <p>Type of variation expressed using Sequence Ontology or LOINC answer list 48006-1.</p>
</td></tr><tr><td><a name="extension-geneticsAlleleName"/><a href="extension-geneticsallelename.html">geneticsAlleleName</a></td><td><b>AlleleName</b> : <p>Common name for variant or gene allele.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicSourceClass"/><a href="extension-geneticsgenomicsourceclass.html">geneticsGenomicSourceClass</a></td><td><b>GenomicSource</b> : <p>The genomic class of the variant: Germline for inherited genome, somatic for cancer genome (e.g. DNA from tumor cells), and prenatal for fetal genome. Associated with LOINC answer list: 48002-0.</p>
</td></tr><tr><td><a name="extension-geneticsAllelicState"/><a href="extension-geneticsallelicstate.html">geneticsAllelicState</a></td><td><b>AllelicState</b> : <p>Level of occurance of the DNA variation in relation to the genomic context.</p>
</td></tr><tr><td><a name="extension-geneticsSpecies"/><a href="extension-geneticsspecies.html">geneticsSpecies</a></td><td><b>Species</b> : <p>Supports testing of human, virus, or bacteria.</p>
</td></tr><tr><td><a name="extension-geneticsCIGAR"/><a href="extension-geneticscigar.html">geneticsCIGAR</a></td><td><b>CIGAR</b> : <p>A sequence of of base lengths and the associated operation, used to indicate which bases align (either a match/mismatch) with the reference, are deleted from the reference, and are insertions that are not in the reference.</p>
</td></tr><tr><td><a name="extension-geneticsAssessedCondition"/><a href="extension-geneticsassessedcondition.html">geneticsAssessedCondition</a></td><td><b>AssessedCondition</b> : <p>Used to denote the condition context for genetic testing e.g. to confirm diagnosis or assess risk for developing a condition.</p>
</td></tr><tr><td><a name="extension-geneticsGeneticsComment"/><a href="extension-geneticsgeneticscomment.html">geneticsGeneticsComment</a></td><td><b>GeneticsComment</b> : <p>Text providing additional information about the genetic finding.</p>
</td></tr><tr><td><a name="extension-geneticsPathogenicityInterpretation"/><a href="extension-geneticspathogenicityinterpretation.html">geneticsPathogenicityInterpretation</a></td><td><b>PathogenicityInterp</b> : <p>Assessment of the pathogenisity (degree of deleteriousness) of the genetic finding.</p>
</td></tr><tr><td><a name="extension-geneticsGeneticsLaboratory"/><a href="extension-geneticsgeneticslaboratory.html">geneticsGeneticsLaboratory</a></td><td><b>GeneticsLab</b> : <p>Genetics laboratory identifier.</p>
</td></tr><tr><td><a name="extension-geneticsGeneticsPatientId"/><a href="extension-geneticsgeneticspatientid.html">geneticsGeneticsPatientId</a></td><td><b>GeneticsPtId</b> : <p>Genetics laboratory patient identifier if different.</p>
</td></tr><tr><td><a name="extension-geneticsGeneticsSpecimenId"/><a href="extension-geneticsgeneticsspecimenid.html">geneticsGeneticsSpecimenId</a></td><td><b>GeneticsSpecId</b> : <p>Genetics laboratory specimen identifier if different.</p>
</td></tr></table><a name="search"> </a>
<h3>Search Parameters</h3>
<p>Search parameters defined by this package. See <a href="search.html">Searching</a> for more information about searching in REST, messaging, and services.</p>
<table class="list">
<tr><td><b>Name</b></td><td><b>Type</b></td><td><b>Description</b></td><td><b>Paths</b></td><td><b>Source</b></td></tr>
<tr><td>amino-acid-change</td><td><a href="search.html#string">string</a></td><td>HGVS Protein Change</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-amino-acid-change.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-amino-acid-change.json.html">JSON</a></td></tr>
<tr><td>assessed-condition</td><td><a href="search.html#string">string</a></td><td>Condition assessed by genetic test</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-assessed-condition.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-assessed-condition.json.html">JSON</a></td></tr>
<tr><td>condition-gene</td><td><a href="search.html#string">string</a></td><td>Condition assessed by genetic test and HGNC gene symbol</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-condition-gene.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-condition-gene.json.html">JSON</a></td></tr>
<tr><td>condition-gene-dnavariant</td><td><a href="search.html#string">string</a></td><td>Condition assessed by genetic testing and HGNC gene symbol and HGVS DNA variant</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-condition-gene-dnavariant.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-condition-gene-dnavariant.json.html">JSON</a></td></tr>
<tr><td>dna-variant</td><td><a href="search.html#string">string</a></td><td>HGVS DNA variant</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-dna-variant.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-dna-variant.json.html">JSON</a></td></tr>
<tr><td>gene-amino-acid-change</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol and HGVS Protein change</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-gene-amino-acid-change.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-gene-amino-acid-change.json.html">JSON</a></td></tr>
<tr><td>gene-dnavariant</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol and HGVS DNA Variant</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-gene-dnavariant.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-gene-dnavariant.json.html">JSON</a></td></tr>
<tr><td>gene-identifier</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol and identifier</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-gene-identifier.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-gene-identifier.json.html">JSON</a></td></tr>
<tr><td>genetic-lab</td><td><a href="search.html#string">string</a></td><td>Genetics laboratory identifier</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-genetic-lab.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-genetic-lab.json.html">JSON</a></td></tr>
<tr><td>genetic-patient</td><td><a href="search.html#string">string</a></td><td>Genetics laboratory patient identifier, if different</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-genetic-patient.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-genetic-patient.json.html">JSON</a></td></tr>
<tr><td>genetic-specimen</td><td><a href="search.html#string">string</a></td><td>Genetics laboratory specimen identifier, if different</td><td></td><td><a href="observation-genetics-cg-prf-1a-Observation-genetic-specimen.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-genetic-specimen.json.html">JSON</a></td></tr>
</table>




</div>


				</div><!-- /inner-wrapper -->
            </div><!-- /row -->
        </div><!-- /container -->
        
    </div><!-- /segment-content -->


	<div id="segment-footer" class="segment"><!-- segment-footer -->
		<div class="container"><!-- container -->
			<div class="inner-wrapper">
				<p>
        © HL7.org 2011+. FHIR DSTU (v0.4.0-4833) generated on Tue, Mar 24, 2015 00:01+0000. <!-- <a href="qa.html">QA Report</a> --> <!-- achive note -->
        <br/>
        <span style="color: #FFFF77">
        Links: <a style="color: #81BEF7" href="timelines.html">What's a DSTU?</a> | 
               <a style="color: #81BEF7" href="history.html">Version History</a> | 
               <a style="color: #81BEF7" href="toc.html">Specification Map</a> | 
               <a style="color: #81BEF7" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Fimplement%2Fstandards%2Ffhir%2Fobservation-genetics-cg-prf-1a&amp;doc2=http%3A%2F%2Fhl7-fhir.github.io%2F%2Fobservation-genetics-cg-prf-1a">Compare to DSTU</a> | 
               <a style="color: #81BEF7" rel="license" href="license.html"><img src="cc0.png" style="border-style: none;" alt="CC0"/></a> | 
               <a style="color: #81BEF7" target="_blank" href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemAdd&amp;tracker_id=677">Propose a change</a>   
        </span>
        </p>
			</div><!-- /inner-wrapper -->
		</div><!-- /container -->
	</div><!-- /segment-footer -->
<!-- disqus thread -->
<!--disqus-->
<!-- end disqus -->        
	
	<div id="segment-post-footer" class="segment hidden"><!-- segment-post-footer -->
		<div class="container"><!-- container -->
		</div><!-- /container -->
	</div><!-- /segment-post-footer -->
    
    <!-- JS and analytics only. -->
    <!-- Bootstrap core JavaScript
================================================== -->
<!-- Placed at the end of the document so the pages load faster -->
<script src="./assets/js/jquery.js"> </script>   <!-- note keep space here, otherwise it will be transformed to empty tag -> fails -->
<script src="./dist/js/bootstrap.min.js"> </script>
<script src="./assets/js/respond.min.js"> </script>

<script src="./assets/js/fhir.js"> </script>

<!-- Analytics Below
================================================== -->

    
</body>
</html>
